src/DynamoCore/Search/SearchElements/CustomNodeSearchElement.cs

```diff
@@ -30,6 +30,26 @@ private set
             }
         }
 
+        public string CustomizationAssembly
+        {
+            get
+            {
+                string customizationPath = System.IO.Path.GetDirectoryName(Path);
+                customizationPath = System.IO.Directory.GetParent(customizationPath).ToString();
+                customizationPath = customizationPath + Dynamo.UI.Configurations.PackageCustomizationAssemblyPostfix;
+                return customizationPath;
```

Comment:
Please use `Directory.GetParent(customizationPath).FullName` instead of `ToString`.